app/javascript/courses/curricula/types/CoursesCurriculum__LevelUpEligibility.res

```diff
@@ -46,5 +47,8 @@ let makeOptionFromJs = js => Belt.Option.map(js, eligibility =>
     | #PreviousLevelIncomplete => PreviousLevelIncomplete
     | #TeamMembersPending => TeamMembersPending
     | #DateLocked => DateLocked
+    | #FutureAddedValue(string) =>
+      Rollbar.error("Unexpected eligibility encountered: " ++ string)
+      raise(UnexpectedEligibilityString(string))
```

Comment:
Is this even required? It's guaranteed that our `graph` server will only send values of the type that we have whitelisted on the server. 

When we add a new value, the complier will automatically error out, saying we haven't handled it yet. I feel like adding this will not have any impact for us.